src/infrastructure/validation/testing/ITest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * @author - Abhishek Saran(111701001)
+ * @module - Infrastucture module 
+ * @team - Test Harness
+ * @description - ITest interface with abstract class TestCase 
+ * @summary - Modules have to extend the abstract class to implement the run method for each test case  
+ */
+
+
+interface ITest{
```

Comment:
Put ITest and TestCase in different files. They are separate files in the package.